cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -368,8 +435,8 @@ void ImageFlip::pcCallback(const sensor_msgs::PointCloud2::ConstPtr& point_cloud
 	if (rotation_angle==0. || rotation_angle==360. || rotation_angle==-360.)
 	{
 		memcpy(&(point_cloud_out_msg->data[0]), &(point_cloud_msg->data[0]), point_cloud_msg->height*point_cloud_msg->width*element_size);
-		rot_mat.at<double>(0,0) = 1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.